### PR TITLE
Colorize important ponyup-init.sh output

### LINKED ISSUE #77: Colorize important messages from ponyup-init.sh
At the moment, there is a lot of output and important messages like "unable to determine libc" and "add this to your PATH" are easy to miss.

We should colorize them to make them stand out more. I would say red for errors, and some other color (blue?) for the informational PATH message.

### PR DESCRIPTION
**Pull request by SeanTAllen**
This should make important messages stand out more.

Closes #77

### CODE REVIEW

**Comment by SeanTAllen** [comment]
I'll update this to use printf instead of echo -e

File: ponyup-init.sh [hunk 1]
```diff
@@ -77,9 +82,10 @@ Linux*)
       platform_triple="${platform_triple}-musl"
       ;;
     *)
-      echo "Unable to determine libc type."
-      echo "If you are using a musl libc based Linux, you'll need to use"
-      echo "--platform=musl when installing ponyc."
+      printf "%bUnable to determine libc type.\n" "${BLUE}"
+      printf "If you are using a musl libc based Linux, you'll need to use\n"
+      printf "%b--platform=musl%b when installing ponyc.%b\n" \
+        "${YELLOW}" "$BLUE}" "${DEFAULT}"
```
Comment (EpicEric) [inline ponyup-init.sh]:
```suggestion
        "${YELLOW}" "${BLUE}" "${DEFAULT}"
```

Comment (SeanTAllen) [reply ponyup-init.sh]: Can you open a PR @EpicEric ?

Comment (EpicEric) [reply ponyup-init.sh]: Sure thing. Opened #83.

**Pull request merged**